apps/dashboard/app/apps/manifest.rb

```diff
@@ -65,6 +65,10 @@ def initialize(opts)
     raise InvalidContentError.new unless(opts && opts.respond_to?(:to_h))
 
     @manifest_options = opts.to_h.with_indifferent_access
+
+    @manifest_options.select! do |method| 
```

Comment:
Why not continue to chain? mutating a member variable a second time seems like a code smell for Ruby.

```suggestion
    @manifest_options = opts.to_h.with_indifferent_access.select! do |method| 
```